tests/test_integration.py

```diff
@@ -78,4 +131,7 @@ def test_integration(recipe_id: str, confpath: str):
         "JOB_NAME": f"{recipe_id}-{str(int(time.time()))}",
     }
     proc = subprocess.run(cmd, capture_output=True, env=env)
-    assert proc.returncode == 0
+    if proc.returncode != 0:
```

Comment:
looks like I botched this. so either:
1. put back the `assert proc.returncode == 0` 
2. make sure `.run(..., text=True)
3. make sure `proc.stdout|stderr` get `.decode()`